storage/innobase/xtrabackup/doc/source/innobackupex/partial_backups_innobackupex.rst

```diff
@@ -64,8 +64,7 @@ Using :option:`innobackupex --databases`
 --------------------------------------------------------------------------------
 
 This option accepts either a space-separated list of the databases and tables to
-backup - in the ``databasename[.tablename]`` form - or a file containing the
```

Comment:
there is an option ---databases-file exists in 2.4 similar to 8.0. Please mention that 


--databases-file specifies a file that can contain multiple databases and tables in the databasename[.tablename] form, one element name per line in the file. Only named databases and tables will be backed up. Names are matched exactly, case-sensitive, with no pattern or regular expression matching.

Reply:
Added.